terraform-provider-constellation/internal/data/providerdata.go

```diff
@@ -0,0 +1,10 @@
+/*
+Copyright (c) Edgeless Systems GmbH
+
+SPDX-License-Identifier: AGPL-3.0-only
+*/
+package data
```

Comment:
```suggestion
*/

package data
```
nit: missing newline